src/AppRoutes.js

```diff
@@ -45,9 +61,9 @@ const AppRoutes = (props) => {
               )
             }
           >
-            {tab.to === '/scenario' && (
+            {(tab.to === 'scenario' || 'instance') && (
```

Comment:
Not sure this syntax works as expected, you may want to replace it by :
`tab.to === 'scenario' || tab.to === 'instance'`
or
`['instance', 'scenario'].includes(tab.to)`

Otherwise, the OR in the condition will consider that `'instance'` is _truthy_ and will return TRUE